### ISSUE TITLE
php 7.4 + phpredis 5 support

### ISSUE THREAD

**Issue by fliespl** [snc/SncRedisBundle#583]
Hello,
I would like to ask about supported versions matrix. Tried to finally change predis to phpredis, but it failed on our setup due to compatibility issues. We have php 7.4 + phpredis 5.2.1.

I.e.
```
  [Symfony\Component\Debug\Exception\ContextErrorException]                                                                                                                                                                            
  Warning: Declaration of Snc\RedisBundle\Client\Phpredis\Client::zAdd($key, $score1, $value1, $score2 = NULL, $value2 = NULL, $scoreN = NULL, $valueN = NULL) should be compatible with Redis::zAdd($key, $score, $value, ...$extra_  
  args)            

```

If phpredis 5.X is not supported maybe it's worth doing an exception on loadPredisClient method with some compaibility issues message?

**Comment by B-Galati** [comment]
Hello, 

A matrix would be a good idea indeed.

You can disable the `logger` to get it working as expected.

**Comment by tf-kevin** [comment]
> You can disable the `logger` to get it working as expected.

Even with disabled logging I get the same exception. 

PHP version 7.4.8
phpredis version 5.3.1 
SncRedisBundle 3.2.3

`Warning: Declaration of Snc\RedisBundle\Client\Phpredis\Client::zAdd($key, $score1, $value1, $score2 = NULL, $value2 = NULL, $scoreN = NULL, $valueN = NULL) should be compatible with Redis::zAdd($key, $score, $value, ...$extra_args)`

**Comment by B-Galati** [comment]
Thanks for the feedback!

**Comment by divine** [comment]
> Thanks for the feedback!

Not sure why you're guys delaying this fix? What's the problem behind it? Missing PR? It's already here?

Thanks for reviewing!

**Comment by yellow1912** [comment]
I guess the author is busy or something. Anyhow, I've just been forced to upgrade doctrine migrations bundle, and one thing lead to another and I found out Doctrine Cache Bundle is deprecated. So I looked into Symfony Cache Component and found out their redis support is also pretty decent. I'm right now in the process of migrating everything to Symfony Cache (we have huge app with many bundles and files), but so far I have moved 50% of the code (I keep Doctrine Cache and SncRedis running for now) and everything is still fine.

You guys should consider switching, if anything it is officially supported by Symfony team which is great.

https://symfony.com/doc/current/components/cache/adapters/redis_adapter.html

The adapter supports Redis, RedisArray, RedisCluster, or Predis. The ability to utilize a cluster of servers to provide redundancy and/or fail-over is also available.
